code/modules/projectiles/guns/projectile/automatic/z8.dm

```diff
@@ -1,36 +1,34 @@
 /obj/item/weapon/gun/projectile/automatic/z8
-	name = "FS AR 5.56x45mm \"Z8 Bulldog\""
-	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Uses 5.56mm rounds."
+	name = "FS CAR 5.56x45mm \"Z8 Bulldog\""
+	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Has underbarrel grenade launcher which are compitable with most modern grenade types. Uses 5.56mm rounds."
 	icon_state = "carbine"
 	item_state = "z8carbine"
 	w_class = ITEM_SIZE_LARGE
 	force = WEAPON_FORCE_PAINFUL
 	caliber = "a556"
 	origin_tech = list(TECH_COMBAT = 8, TECH_MATERIAL = 3)
 	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 6)
-	price_tag = 3500
+	price_tag = 3200 //old but gold, decent AP caliber, underbarrel GL, mild recoil and 20-round mags. Better than FS AK.
 	ammo_type = "/obj/item/ammo_casing/a556"
 	fire_sound = 'sound/weapons/guns/fire/batrifle_fire.ogg'
 	slot_flags = SLOT_BACK
 	load_method = MAGAZINE
 	mag_well = MAG_WELL_CIVI_RIFLE
 	magazine_type = /obj/item/ammo_magazine/a556
-	auto_eject = 1
```

Comment:
Why you remove this line when the Z8 has auto-ejecting audio with it originally.
Do you plan also remove auto_eject_sound ?
If not, just copy audio line and put with all guns where the auto_eject was left enabled.

Reply:
Will do, thanks.